script.js

```diff
@@ -1,12 +1,1882 @@
+      airtime: "21:00",
+      airstamp: "2019-04-29T01:00:00+00:00",
+      runtime: 90,
+      image: {
+        medium:
+          "http://static.tvmaze.com/uploads/images/medium_landscape/192/482465.jpg",
+        original:
+          "http://static.tvmaze.com/uploads/images/original_untouched/192/482465.jpg",
+      },
+      summary: "<p>Winterfell fights the Army of the Dead.</p>",
+      _links: {
+        self: {
+          href: "http://api.tvmaze.com/episodes/1623965",
+        },
+      },
+    },
+    {
+      id: 1623966,
+      url: "http://www.tvmaze.com/episodes/1623966/game-of-thrones-8x04-the-last-of-the-starks",
+      name: "The Last of the Starks",
+      season: 8,
+      number: 4,
+      airdate: "2019-05-05",
+      airtime: "21:00",
+      airstamp: "2019-05-06T01:00:00+00:00",
+      runtime: 78,
+      image: {
+        medium:
+          "http://static.tvmaze.com/uploads/images/medium_landscape/195/487839.jpg",
+        original:
+          "http://static.tvmaze.com/uploads/images/original_untouched/195/487839.jpg",
+      },
+      summary:
+        "<p>The survivors plan their next steps; Cersei makes a power move.</p><p></p>",
+      _links: {
+        self: {
+          href: "http://api.tvmaze.com/episodes/1623966",
+        },
+      },
+    },
+    {
+      id: 1623967,
+      url: "http://www.tvmaze.com/episodes/1623967/game-of-thrones-8x05-the-bells",
+      name: "The Bells",
+      season: 8,
+      number: 5,
+      airdate: "2019-05-12",
+      airtime: "21:00",
+      airstamp: "2019-05-13T01:00:00+00:00",
+      runtime: 79,
+      image: {
+        medium:
+          "http://static.tvmaze.com/uploads/images/medium_landscape/196/491994.jpg",
+        original:
+          "http://static.tvmaze.com/uploads/images/original_untouched/196/491994.jpg",
+      },
+      summary:
+        "<p>Varys betrays his queen, and Daenerys brings her forces to King's Landing.</p>",
+      _links: {
+        self: {
+          href: "http://api.tvmaze.com/episodes/1623967",
+        },
+      },
+    },
+    {
+      id: 1623968,
+      url: "http://www.tvmaze.com/episodes/1623968/game-of-thrones-8x06-the-iron-throne",
+      name: "The Iron Throne",
+      season: 8,
+      number: 6,
+      airdate: "2019-05-19",
+      airtime: "21:00",
+      airstamp: "2019-05-20T01:00:00+00:00",
+      runtime: 80,
+      image: {
+        medium:
+          "http://static.tvmaze.com/uploads/images/medium_landscape/198/495648.jpg",
+        original:
+          "http://static.tvmaze.com/uploads/images/original_untouched/198/495648.jpg",
+      },
+      summary:
+        "<p>In the aftermath of the devastating attack on King's Landing, Daenerys must face the survivors.</p>",
+      _links: {
+        self: {
+          href: "http://api.tvmaze.com/episodes/1623968",
+        },
+      },
+    },
+  ];
+}
+
+let getEpi = getAllEpisodes();
+
+function getEpiName(getEpi) {
+  for (let i = 0; i < getEpi.length; i++) {
```

Comment:
Very nice for loop. Do you think there's a more efficient way we could write this?